src/quacc/atoms/skzcam.py

```diff
@@ -251,112 +342,10 @@ def get_ecp_region(
                 if (
                     dist < ecp_dist
                     and idx not in dummy_cation_idx
-                    and emb_clus.get_array("atom_type")[idx] == "cation"
+                    and embedded_cluster.get_array("atom_type")[idx] == "cation"
                 ):
                     cluster_ecp_region_idx += [idx]
 
         ecp_region_idx += [list(set(cluster_ecp_region_idx))]
 
     return ecp_region_idx
-
-
-def create_skzcam_clusters(
```

Comment:
I moved this function to the top of the file since it is the one that the users are interested in.